extension/manifest.json

```diff
@@ -41,6 +47,15 @@
     }
   },
   "background": {
-    "service_worker": "background.js"
+    "service_worker": "background.js",
+    "scripts": [
+      "background.js"
+    ]
+  },
+  "browser_specific_settings": {
```

Comment:
Is this needed for Firefox in `manifest.json`?

Reply:
yes, it is helpful for telling firefox to compatible with manifest v3.

ref.

[stackoverflow.com/questions/78491335/how-can-i-write-a-chrome-and-firefox-manifest-v3-extension-that-shares-code-when](https://stackoverflow.com/questions/78491335/how-can-i-write-a-chrome-and-firefox-manifest-v3-extension-that-shares-code-when)